lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -193,7 +194,7 @@ public IJobSubmissionResult SubmitAndGetJobStatus(JobRequest jobRequest)
         /// Return current Job status
         /// </summary>
         /// <returns></returns>
-        /// TODO: REEF-889
+        /// TODO: [REEF-889]
```

Comment:
Yes. This adds YARN-only APIs to the generic REEF client. This will be hart (and maybe impossible) to support in the long run.